packages/test-harness/scripts/build-tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TESTS="$(find .. -name '*.test.ts')"
+
+pnpm run build:base --outdir=dist --out-extension:.js=.cjs $TESTS
```

Comment:
The only thing I'm not entirely happy with about this PR is this line. This results in about 100 MBs of bundled output, because each test bundles up all of its dependencies. I couldn't come up with a workaround because our tests run in a fairly constrained environment. We can't customize the loader that node uses because we are running within VS Code, so node's loader has already been configured because it's configured at launch time.